core/src/mindustry/graphics/OverlayRenderer.java

```diff
@@ -171,7 +171,7 @@ public void drawTop(){
             Draw.reset();
 
             Building tile = world.buildWorld(v.x, v.y);
-            if(input.canDropItem() && tile != null && tile.interactable(player.team()) && tile.acceptStack(player.unit().item(), player.unit().stack.amount, player.unit()) > 0 && player.within(tile, itemTransferRange)){
+            if(input.canDropItem() && tile != null && tile.interactable(player.team()) && (tile.acceptStack(player.unit().item(), player.unit().stack.amount, player.unit()) > 0 || player.unit().formation != null) && player.within(tile, itemTransferRange)){
```

Comment:
This is buggy. I can be in a formation with no items, and yet the unload cursor will still display and I will be able to drag a non-existent item to a container.

If the player has no items and tries to drag a formation member's item into a container, the item displayed will be the last item the player used, not the actual item that will be deposited.